src/documents/LocalizationInThePreferencesFramework.md

```diff
@@ -32,51 +34,70 @@ Message bundles cannot contain arrays. Instead a namespace should be used to gro
 
-- parses and assembles JSON strings loaded by the MessageLoader,
-- runs the assembled JSON through the [message resolver](http://wiki.fluidproject.org/display/docs/fluid.messageResolver) to create the lookup function, and
-- attaches the message resolver bundle as a member, accessible through `that.msgResolver`.
+For a `fluid.prefs.panel` component all of the `fluid.prefs.msgLookup` configuration is prewired. For other components to make use of this functionality the following configuration is needed:
 
-To access the message bundle from other components on the IoC tree, use `{prefsEditorLoader}.msgResolver`.
+- add `fluid.prefs.msgLookup` as a [parent grade](ComponentGrades.md#specifying-parent-grades)
+- add an instance of `fluid.messageResolver` as a subcomponent with the member name `msgResolver`
+- link the messageBase to the fetched Message Bundle.
+    - e.g. using an IoC reference to the [messageLoader](#messageloader): `{messageLoader}.resources.<messageBundle>.resourceText`
+- for an instance of `fluid.rendererComponent` it is also necessary to use the messageResolver's `resolve` method as the `messageLocator`
 
-### PrefsEditor
-
-PrefsEditor is the host component that holds all the actual panel (or adjuster) components as subcomponents. By default, the message bundle is *not* passed down to PrefsEditor. If your PrefsEditor component will need direct access to the message bundle, provide it at the instantiation of any PrefsEditor instance, as shown in the following example:
+#### Adding MsgLookup to a Component
 
 ```javascript
-fluid.prefs.separatedPanel("#myPrefsEditor", {
-    prefsEditor: {
-        msgResolver: "{prefsEditorLoader}.msgResolver"
+fluid.defaults("my.component", {
+    gradeNames: ["fluid.prefs.msgLookup", "fluid.component"],
+    components: {
+        msgResolver: {
+            type: "fluid.messageResolver"
+        }
+    },
+    // should contain the strings loaded from a Message Bundle
+    // e.g. "{messageLoader}.resources.messageBundleName.resourceText"
+    messageBase: {},
+    distributeOptions: {
+        source: "{that}.options.messageBase",
+        target: "{that > msgResolver}.options.messageBase"
     }
 });
 ```
 
-If the message bundle is provided to PrefsEditor this way, access it within the PrefsEditor component using `{that}.options.msgResolver`.
+#### Adding MsgLookup to a Renderer Component
 
-### Panels
-
-The message bundle is attached to each panel component as the `parentBundle` option. To access it from within a panel, use `{that}.options.parentBundle`.
+```javascript
+fluid.defaults("my.renderer.component", {
+    gradeNames: ["fluid.prefs.msgLookup", "fluid.rendererComponent"],
+    components: {
```

Comment:
Yes it seems so. https://github.com/fluid-project/infusion/blob/master/src/framework/preferences/js/Panels.js#L59-L82